--- Celestia-1.6.2/src/celestia/gtk/data/celestia.desktop~	2020-11-06 23:34:57.000000000 +0000
+++ Celestia-1.6.2/src/celestia/gtk/data/celestia.desktop	2021-08-17 07:09:49.377162127 +0000
@@ -9,5 +9,7 @@
 Name=Celestia (Gtk+)
 GenericName=Space Simulator
 Comment=Explore the Universe in this detailed space simulation
+Comment[pl]=Symulacja przestrzeni kosmicznej w czasie rzeczywistym
 StartupNotify=true
-Categories=Astronomy;Education;Application;
+Categories=Qt;KDE;Education;Science;Astronomy;
+# vi: encoding=utf-8
--- Celestia-1.6.2/src/celestia/kde/data/Makefile.am~	2020-11-06 23:34:57.000000000 +0000
+++ Celestia-1.6.2/src/celestia/kde/data/Makefile.am	2021-08-17 07:12:45.142135370 +0000
@@ -14,7 +14,7 @@
 mimedir = $(kde_mimedir)/application
 mime_DATA = x-celestia-script.desktop
 
-appsdir = $(kde_appsdir)/Edutainment/Science
+appsdir = $(appsdir)
 apps_DATA = celestia.desktop
 
 KDE_ICON = celestia
--- Celestia-1.6.2/src/celestia/kde/data/celestia.desktop~	2020-11-06 23:34:57.000000000 +0000
+++ Celestia-1.6.2/src/celestia/kde/data/celestia.desktop	2021-08-17 07:15:26.860041315 +0000
@@ -1,8 +1,14 @@
 [Desktop Entry]
+Encoding=UTF-8
 Name=Celestia
-Comment=Celestia
+Comment=A real-time visual space simulation
+Comment[pl]=Symulacja przestrzeni kosmicznej w czasie rzeczywistym
 Exec=celestia %f
 Terminal=false
 Icon=celestia.png
 MimeType=application/x-celestia-script
 Type=Application
+Categories=Qt;KDE;Education;Science;Astronomy;
+DocPath=celestia/index.html
+StartupNotify=true
+# vi: encoding=utf-8
