--- configure.in~	2008-09-02 19:01:47.000000000 +0100
+++ configure.in	2008-09-02 19:09:09.000000000 +0100
@@ -399,7 +399,7 @@
 
 if (test "$enable_lua" != "no"); then
 	LUA_VER=0
-	PKG_CHECK_MODULES(LUA, lua5.1 >= 5.1.0, LUA_VER=0x050100, 
+	PKG_CHECK_MODULES(LUA, lua51 >= 5.1.0, LUA_VER=0x050100, 
 		[ PKG_CHECK_MODULES(LUA, lua >= 5.1.0, LUA_VER=0x050100, 
 		  [ PKG_CHECK_MODULES(LUA, lua50 >= 5.0.0 lua50 < 5.1.0, LUA_VER=0x050000,
 		    [ PKG_CHECK_MODULES(LUA, lua >= 5.0.0 lua < 5.1.0, LUA_VER=0x050000, AC_MSG_RESULT([no])) ] 
