diff -ruN celestia../src/celengine/render.cpp celestia./src/celengine/render.cpp
--- celestia../src/celengine/render.cpp 2004-07-08 22:53:35.000000000 +0200
+++ celestia./src/celengine/render.cpp  2004-07-09 16:27:28.333832376 +0200
@@ -152,7 +152,7 @@
     locationFilter(~0),
     distanceLimit(1.0e6f)
 {
-    starVertexBuffer = new StarVertexBuffer(2048);
+    starVertexBuffer = new StarVertexBuffer(1);
     skyVertices = new SkyVertex[MaxSkySlices * (MaxSkyRings + 1)];
     skyIndices = new uint32[(MaxSkySlices + 1) * 2 * MaxSkyRings];
     skyContour = new SkyContourPoint[MaxSkySlices + 1];