diff -dur -x '*~' -x '*.rej' -x '*.orig' celestia-1.6.1.orig/src/celestia/gtk/data/celestia.desktop celestia-1.6.1/src/celestia/gtk/data/celestia.desktop
--- celestia-1.6.1.orig/src/celestia/gtk/data/celestia.desktop	2011-06-05 18:11:10.000000000 +0200
+++ celestia-1.6.1/src/celestia/gtk/data/celestia.desktop	2015-07-10 19:01:11.534166719 +0200
@@ -1,12 +1,13 @@
 [Desktop Entry]
 Encoding=UTF-8
-Version=1.4.0
 Type=Application
 Exec=celestia
 Icon=celestia
 X-GNOME-DocPath=
 Terminal=false
 Name=Celestia
-Comment=Space Simulator
+Comment=A real-time visual space simulation
+Comment[pl]=Symulacja przestrzeni kosmicznej w czasie rzeczywistym
 StartupNotify=true
-Categories=Astronomy;Science;Application;
+Categories=Qt;KDE;Education;Science;Astronomy;
+# vi: encoding=utf-8
diff -dur -x '*~' -x '*.rej' -x '*.orig' celestia-1.6.1.orig/src/celestia/kde/data/Makefile.am celestia-1.6.1/src/celestia/kde/data/Makefile.am
--- celestia-1.6.1.orig/src/celestia/kde/data/Makefile.am	2011-06-05 18:11:13.000000000 +0200
+++ celestia-1.6.1/src/celestia/kde/data/Makefile.am	2015-07-10 18:58:02.449884263 +0200
@@ -14,7 +14,7 @@
 mimedir = $(kde_mimedir)/application
 mime_DATA = x-celestia-script.desktop
 
-appsdir = $(kde_appsdir)/Edutainment/Science
+appsdir = $(appsdir)
 apps_DATA = celestia.desktop
 
 KDE_ICON = celestia
diff -dur -x '*~' -x '*.rej' -x '*.orig' celestia-1.6.1.orig/src/celestia/kde/data/celestia.desktop celestia-1.6.1/src/celestia/kde/data/celestia.desktop
--- celestia-1.6.1.orig/src/celestia/kde/data/celestia.desktop	2011-06-05 18:11:13.000000000 +0200
+++ celestia-1.6.1/src/celestia/kde/data/celestia.desktop	2015-07-10 18:58:59.646838368 +0200
@@ -1,8 +1,14 @@
 [Desktop Entry]
+Encoding=UTF-8
 Name=Celestia
-Comment=Celestia
+Comment=A real-time visual space simulation
+Comment[pl]=Symulacja przestrzeni kosmicznej w czasie rzeczywistym
 Exec=celestia %f
 Terminal=false
 Icon=celestia.png
 MimeType=application/x-celestia-script
 Type=Application
+Categories=Qt;KDE;Education;Science;Astronomy;
+DocPath=celestia/index.html
+StartupNotify=true
+# vi: encoding=utf-8
