diff -urN celestia-1.4.1.orig/configure.in celestia-1.4.1.mod/configure.in
--- celestia-1.4.1.orig/configure.in	2006-02-03 19:09:54.000000000 +0100
+++ celestia-1.4.1.mod/configure.in	2006-06-29 21:41:44.159248250 +0200
@@ -256,8 +256,9 @@
 dnl (this is a silly trick to make configure behave)
 AM_CONDITIONAL(GCONF_SCHEMAS_INSTALL, test "x" = "y")
 
+PKG_PROG_PKG_CONFIG
+
 if (test "$ui_gtk" != "no"); then
-	PKG_PROG_PKG_CONFIG
 
 	dnl GNOME is an extension to the GTK options
 	if (test "$ui_gnome" = "yes"); then
@@ -363,43 +364,15 @@
 
 
 AC_ARG_WITH([lua],
-            AC_HELP_STRING([--with-lua[=DIR]],
+            AC_HELP_STRING([--with-lua],
                            [Use Lua for Celestia Extension Language support]),
-            enable_lua="$withval",
+            enable_lua="yes",
             enable_lua="no")
 			
-AC_ARG_WITH([lua-inc],
-            AC_HELP_STRING([--with-lua-includes[=DIR]],
-                           [Specify location of Lua headers]),
-            lua_includes="$withval",
-            lua_includes="no")
-			
-AC_ARG_WITH([lua-lib],
-            AC_HELP_STRING([--with-lua-libs[=DIR]],
-                           [Specify location of Lua libs]),
-            lua_libs="$withval",
-            lua_libs="no")
-
-AC_MSG_CHECKING([whether to enable Celestia Extension Language])
-if (test "$enable_lua" != "no"); then
-	CXXFLAGS="$CXXFLAGS -DCELX"
-	LDFLAGS="$LDFLAGS -llualib -llua"
-	
-	if (test "$lua_includes" != "no"); then
-		CXXFLAGS="$CXXFLAGS -I$lua_includes"
-	fi
-	
-	if (test "$lua_libs" != "no"); then
-		LIBS="$LIBS -L$lua_libs"
-	fi
-	AC_MSG_RESULT(yes)
-	
-	if (test "$enable_lua" != "yes"); then
-		CXXFLAGS="$CXXFLAGS -I$enable_lua/include"
-		LIBS="$LIBS  -L$enable_lua/lib"
-	fi
-else
-	AC_MSG_RESULT(no)
+if (test "x$enable_lua" = "xyes"); then
+	PKG_CHECK_MODULES(LUA, lua,
+			  CXXFLAGS="$CXXFLAGS -DCELX $LUA_CFLAGS"
+			  LIBS="$LIBS $LUA_LIBS")
 fi
 AM_CONDITIONAL(ENABLE_CELX, test "$enable_lua" != "no")
 
