diff -dur celestia-1.6.1.orig/src/celestia/Makefile.am celestia-1.6.1/src/celestia/Makefile.am
--- celestia-1.6.1.orig/src/celestia/Makefile.am	2011-06-05 18:11:13.000000000 +0200
+++ celestia-1.6.1/src/celestia/Makefile.am	2015-07-10 18:49:02.940315395 +0200
@@ -7,14 +7,14 @@
 
 if ENABLE_KDE
 SUBDIRS += kde
-celestiaKDELIBS = $(LIB_QT) $(LIB_KDECORE) $(LIB_KDEUI) $(LIB_KFILE) \
-                  $(LIBSOCKET) -lDCOP kde/libkdegui.a
+celestiaKDELIBS = kde/libkdegui.a $(LIB_QT) $(LIB_KDECORE) $(LIB_KDEUI) \
+		  $(LIB_KFILE) $(LIBSOCKET) -lDCOP
 celestia_LDFLAGS = $(all_libraries) $(KDE_RPATH)
 endif
 
 if ENABLE_GTK
 SUBDIRS += gtk
-celestiaGTKLIBS = $(GTK_LIBS) gtk/libgtkgui.a
+celestiaGTKLIBS = gtk/libgtkgui.a $(GTK_LIBS)
 endif
 
 COMMONSOURCES = \
@@ -88,7 +88,8 @@
 	../cel3ds/libcel3ds.a \
 	../celmath/libcelmath.a \
 	../celutil/libcelutil.a \
-	$(SPICE_LIBS)
+	$(SPICE_LIBS) \
+	$(GL_LIBS)
 
 noinst_HEADERS = $(wildcard *.h)
 noinst_DATA = ../../celestia
diff -dur celestia-1.6.1.orig/src/celestia/Makefile.in celestia-1.6.1/src/celestia/Makefile.in
--- celestia-1.6.1.orig/src/celestia/Makefile.in	2011-06-05 20:34:28.000000000 +0200
+++ celestia-1.6.1/src/celestia/Makefile.in	2015-07-10 18:49:02.940315395 +0200
@@ -473,11 +473,11 @@
 xdg_menudir = @xdg_menudir@
 SUBDIRS = res $(am__append_1) $(am__append_2)
 INCLUDES = -I..
-@ENABLE_KDE_TRUE@celestiaKDELIBS = $(LIB_QT) $(LIB_KDECORE) $(LIB_KDEUI) $(LIB_KFILE) \
-@ENABLE_KDE_TRUE@                  $(LIBSOCKET) -lDCOP kde/libkdegui.a
+@ENABLE_KDE_TRUE@celestiaKDELIBS = kde/libkdegui.a $(LIB_QT) $(LIB_KDECORE) $(LIB_KDEUI) \
+@ENABLE_KDE_TRUE@                  $(LIB_KFILE) $(LIBSOCKET) -lDCOP
 
 @ENABLE_KDE_TRUE@celestia_LDFLAGS = $(all_libraries) $(KDE_RPATH)
-@ENABLE_GTK_TRUE@celestiaGTKLIBS = $(GTK_LIBS) gtk/libgtkgui.a
+@ENABLE_GTK_TRUE@celestiaGTKLIBS = gtk/libgtkgui.a $(GTK_LIBS)
 COMMONSOURCES = \
 	celestiacore.cpp \
 	configfile.cpp \
@@ -538,7 +538,8 @@
 	../cel3ds/libcel3ds.a \
 	../celmath/libcelmath.a \
 	../celutil/libcelutil.a \
-	$(SPICE_LIBS)
+	$(SPICE_LIBS) \
+	$(GL_LIBS)
 
 noinst_HEADERS = $(wildcard *.h)
 noinst_DATA = ../../celestia
