diff -ruN celestia-1.2.4-/data/solarsys.ssc celestia-1.2.4/data/solarsys.ssc
--- celestia-1.2.4-/data/solarsys.ssc	Tue Apr 30 19:36:02 2002
+++ celestia-1.2.4/data/solarsys.ssc	Thu Jan  2 21:50:53 2003
@@ -285,6 +285,12 @@
                              # calculations of satellite positions
 	                     # old values: 337.68, 157.68
 	Albedo            0.51
+
+	Rings {
+		Inner 92000
+		Outer 221000
+		Texture "jupiter-rings.png"
+	}
 }
 
 "Amalthea" "Sol/Jupiter"
@@ -784,6 +790,12 @@
 	LongOfRotationAxis 49.235
 
 	Albedo            0.62
+
+	Rings {
+		Inner 41900
+		Outer 62983
+		Texture "neptune-rings.png"
+	}
 }
 
 "Proteus" "Sol/Neptune"
