diff -dur -x '*~' celestia-1.6.1.orig/src/celengine/asterism.cpp celestia-1.6.1/src/celengine/asterism.cpp
--- celestia-1.6.1.orig/src/celengine/asterism.cpp	2011-06-05 18:11:09.000000000 +0200
+++ celestia-1.6.1/src/celengine/asterism.cpp	2015-07-10 19:59:04.239679001 +0200
@@ -15,6 +15,7 @@
 #endif /* TARGET_OS_MAC */
 #endif /* _WIN32 */
 
+#include <libintl.h>
 #include <celutil/util.h>
 #include <celutil/debug.h>
 #include "parser.h"
diff -dur -x '*~' celestia-1.6.1.orig/src/celengine/astro.cpp celestia-1.6.1/src/celengine/astro.cpp
--- celestia-1.6.1.orig/src/celengine/astro.cpp	2011-06-05 18:11:09.000000000 +0200
+++ celestia-1.6.1/src/celengine/astro.cpp	2015-07-10 19:59:04.243012354 +0200
@@ -11,6 +11,7 @@
 #include <cmath>
 #include <iomanip>
 #include <cstdio>
+#include <cstring>
 #include <time.h>
 #include <celutil/basictypes.h>
 #include <celmath/mathlib.h>
diff -dur -x '*~' celestia-1.6.1.orig/src/celengine/dsodb.cpp celestia-1.6.1/src/celengine/dsodb.cpp
--- celestia-1.6.1.orig/src/celengine/dsodb.cpp	2011-06-05 18:11:09.000000000 +0200
+++ celestia-1.6.1/src/celengine/dsodb.cpp	2015-07-10 19:59:04.243012354 +0200
@@ -14,6 +14,7 @@
 #include <cstdlib>
 #include <cstdio>
 #include <cassert>
+#include <libintl.h>
 #include <algorithm>
 #include <celmath/mathlib.h>
 #include <celmath/plane.h>
diff -dur -x '*~' celestia-1.6.1.orig/src/celengine/fragmentprog.cpp celestia-1.6.1/src/celengine/fragmentprog.cpp
--- celestia-1.6.1.orig/src/celengine/fragmentprog.cpp	2011-06-05 18:11:09.000000000 +0200
+++ celestia-1.6.1/src/celengine/fragmentprog.cpp	2015-07-10 19:59:04.243012354 +0200
@@ -7,6 +7,7 @@
 // as published by the Free Software Foundation; either version 2
 // of the License, or (at your option) any later version.
 
+#include <libintl.h>
 #include <iostream>
 #include <fstream>
 #include <string>
diff -dur -x '*~' celestia-1.6.1.orig/src/celengine/galaxy.cpp celestia-1.6.1/src/celengine/galaxy.cpp
--- celestia-1.6.1.orig/src/celengine/galaxy.cpp	2011-06-05 18:11:09.000000000 +0200
+++ celestia-1.6.1/src/celengine/galaxy.cpp	2015-07-10 19:59:04.243012354 +0200
@@ -12,6 +12,8 @@
 #include <algorithm>
 #include <cstdio>
 #include <cassert>
+#include <cstring>
+#include <libintl.h>
 #include "celestia.h"
 #include <celmath/mathlib.h>
 #include <celmath/perlin.h>
diff -dur -x '*~' celestia-1.6.1.orig/src/celengine/image.cpp celestia-1.6.1/src/celengine/image.cpp
--- celestia-1.6.1.orig/src/celengine/image.cpp	2011-06-05 18:11:09.000000000 +0200
+++ celestia-1.6.1/src/celengine/image.cpp	2015-07-10 19:59:04.243012354 +0200
@@ -8,6 +8,7 @@
 // of the License, or (at your option) any later version.
 
 #include <fstream>
+#include <cstring>
 
 #ifndef TARGET_OS_MAC
 #define JPEG_SUPPORT
diff -dur -x '*~' celestia-1.6.1.orig/src/celengine/location.cpp celestia-1.6.1/src/celengine/location.cpp
--- celestia-1.6.1.orig/src/celengine/location.cpp	2011-06-05 18:11:09.000000000 +0200
+++ celestia-1.6.1/src/celengine/location.cpp	2015-07-10 19:59:04.243012354 +0200
@@ -7,6 +7,7 @@
 // as published by the Free Software Foundation; either version 2
 // of the License, or (at your option) any later version.
 
+#include <libintl.h>
 #include <map>
 #include <celengine/location.h>
 #include <celengine/body.h>
diff -dur -x '*~' celestia-1.6.1.orig/src/celengine/meshmanager.cpp celestia-1.6.1/src/celengine/meshmanager.cpp
--- celestia-1.6.1.orig/src/celengine/meshmanager.cpp	2011-06-05 18:11:09.000000000 +0200
+++ celestia-1.6.1/src/celengine/meshmanager.cpp	2015-07-10 19:59:04.243012354 +0200
@@ -13,6 +13,7 @@
 #include <iostream>
 #include <fstream>
 #include <cassert>
+#include <libintl.h>
 
 #include "celestia.h"
 #include <celutil/debug.h>
diff -dur -x '*~' celestia-1.6.1.orig/src/celengine/nebula.cpp celestia-1.6.1/src/celengine/nebula.cpp
--- celestia-1.6.1.orig/src/celengine/nebula.cpp	2011-06-05 18:11:09.000000000 +0200
+++ celestia-1.6.1/src/celengine/nebula.cpp	2015-07-10 19:59:04.243012354 +0200
@@ -8,7 +8,8 @@
 // of the License, or (at your option) any later version.
 
 #include <algorithm>
-#include <stdio.h>
+#include <cstdio>
+#include <libintl.h>
 #include "celestia.h"
 #include <celmath/mathlib.h>
 #include <celutil/util.h>
diff -dur -x '*~' celestia-1.6.1.orig/src/celengine/opencluster.cpp celestia-1.6.1/src/celengine/opencluster.cpp
--- celestia-1.6.1.orig/src/celengine/opencluster.cpp	2011-06-05 18:11:09.000000000 +0200
+++ celestia-1.6.1/src/celengine/opencluster.cpp	2015-07-10 19:59:04.243012354 +0200
@@ -8,7 +8,8 @@
 // of the License, or (at your option) any later version.
 
 #include <algorithm>
-#include <stdio.h>
+#include <cstdio>
+#include <libintl.h>
 #include "celestia.h"
 #include <celmath/mathlib.h>
 #include <celutil/util.h>
diff -dur -x '*~' celestia-1.6.1.orig/src/celengine/overlay.cpp celestia-1.6.1/src/celengine/overlay.cpp
--- celestia-1.6.1.orig/src/celengine/overlay.cpp	2011-06-05 18:11:09.000000000 +0200
+++ celestia-1.6.1/src/celengine/overlay.cpp	2015-07-10 19:59:04.243012354 +0200
@@ -10,6 +10,7 @@
 #include <cstring>
 #include <cstdarg>
 #include <cstdio>
+#include <cstring>
 #include <celutil/utf8.h>
 #include "gl.h"
 #include "vecgl.h"
diff -dur -x '*~' celestia-1.6.1.orig/src/celengine/solarsys.cpp celestia-1.6.1/src/celengine/solarsys.cpp
--- celestia-1.6.1.orig/src/celengine/solarsys.cpp	2011-06-05 18:11:09.000000000 +0200
+++ celestia-1.6.1/src/celengine/solarsys.cpp	2015-07-10 19:59:04.243012354 +0200
@@ -19,6 +19,7 @@
 #endif /* ! TARGET_OS_MAC */
 #endif /* ! _WIN32 */
 
+#include <libintl.h>
 #include <celutil/debug.h>
 #include <celmath/mathlib.h>
 #include <celutil/util.h>
diff -dur -x '*~' celestia-1.6.1.orig/src/celengine/stardb.cpp celestia-1.6.1/src/celengine/stardb.cpp
--- celestia-1.6.1.orig/src/celengine/stardb.cpp	2011-06-05 18:11:09.000000000 +0200
+++ celestia-1.6.1/src/celengine/stardb.cpp	2015-07-10 19:59:04.243012354 +0200
@@ -11,7 +11,9 @@
 #include <cmath>
 #include <cstdlib>
 #include <cstdio>
+#include <cstring>
 #include <cassert>
+#include <libintl.h>
 #include <algorithm>
 #include <celmath/mathlib.h>
 #include <celmath/plane.h>
diff -dur -x '*~' celestia-1.6.1.orig/src/celengine/texture.cpp celestia-1.6.1/src/celengine/texture.cpp
--- celestia-1.6.1.orig/src/celengine/texture.cpp	2011-06-05 18:11:09.000000000 +0200
+++ celestia-1.6.1/src/celengine/texture.cpp	2015-07-10 19:59:04.246345706 +0200
@@ -22,12 +22,14 @@
 #endif
 
 #include <cmath>
+#include <libintl.h>
 #include <algorithm>
 #include <iostream>
 #include <fstream>
 #include <cstdlib>
 #include <cstdio>
 #include <cassert>
+#include <cstring>
 
 #ifndef _WIN32
 #ifndef TARGET_OS_MAC
diff -dur -x '*~' celestia-1.6.1.orig/src/celengine/vertexprog.cpp celestia-1.6.1/src/celengine/vertexprog.cpp
--- celestia-1.6.1.orig/src/celengine/vertexprog.cpp	2011-06-05 18:11:09.000000000 +0200
+++ celestia-1.6.1/src/celengine/vertexprog.cpp	2015-07-10 19:59:04.246345706 +0200
@@ -7,6 +7,7 @@
 // as published by the Free Software Foundation; either version 2
 // of the License, or (at your option) any later version.
 
+#include <libintl.h>
 #include <iostream>
 #include <fstream>
 #include <string>
diff -dur -x '*~' celestia-1.6.1.orig/src/celestia/celestiacore.cpp celestia-1.6.1/src/celestia/celestiacore.cpp
--- celestia-1.6.1.orig/src/celestia/celestiacore.cpp	2011-06-05 18:11:13.000000000 +0200
+++ celestia-1.6.1/src/celestia/celestiacore.cpp	2015-07-10 19:59:04.246345706 +0200
@@ -23,6 +23,7 @@
 #include <cstring>
 #include <cassert>
 #include <ctime>
+#include <libintl.h>
 #include <celengine/gl.h>
 #include <celmath/vecmath.h>
 #include <celmath/quaternion.h>
diff -dur -x '*~' celestia-1.6.1.orig/src/celestia/glutmain.cpp celestia-1.6.1/src/celestia/glutmain.cpp
--- celestia-1.6.1.orig/src/celestia/glutmain.cpp	2011-06-05 18:11:13.000000000 +0200
+++ celestia-1.6.1/src/celestia/glutmain.cpp	2015-07-10 19:59:04.246345706 +0200
@@ -15,6 +15,7 @@
 #include <cstdlib>
 #include <cctype>
 #include <cstring>
+#include <libintl.h>
 #include <time.h>
 #include <unistd.h>
 #include <celengine/gl.h>
diff -dur -x '*~' celestia-1.6.1.orig/src/celestia/imagecapture.cpp celestia-1.6.1/src/celestia/imagecapture.cpp
--- celestia-1.6.1.orig/src/celestia/imagecapture.cpp	2011-06-05 18:11:13.000000000 +0200
+++ celestia-1.6.1/src/celestia/imagecapture.cpp	2015-07-10 19:59:25.123132399 +0200
@@ -28,6 +28,7 @@
 #ifdef MACOSX
 #include "../celestia/Celestia.app.skel/Contents/Frameworks/Headers/png.h"
 #else
+#include <zlib.h>
 #include "png.h"
 #endif
 
diff -dur -x '*~' celestia-1.6.1.orig/src/celestia/kde/kdeapp.cpp celestia-1.6.1/src/celestia/kde/kdeapp.cpp
--- celestia-1.6.1.orig/src/celestia/kde/kdeapp.cpp	2011-06-05 18:11:13.000000000 +0200
+++ celestia-1.6.1/src/celestia/kde/kdeapp.cpp	2015-07-10 19:59:04.246345706 +0200
@@ -19,6 +19,7 @@
 #include <libintl.h>
 #include <fstream>
 #include <sstream>
+#include <libintl.h>
 
 #include <qpushbutton.h>
 #include <qslider.h>
diff -dur -x '*~' celestia-1.6.1.orig/src/celestia/kde/kdeuniquecelestia.cpp celestia-1.6.1/src/celestia/kde/kdeuniquecelestia.cpp
--- celestia-1.6.1.orig/src/celestia/kde/kdeuniquecelestia.cpp	2011-06-05 18:11:13.000000000 +0200
+++ celestia-1.6.1/src/celestia/kde/kdeuniquecelestia.cpp	2015-07-10 19:59:04.249679059 +0200
@@ -20,6 +20,7 @@
 #include <qdir.h>
 #include <string>
 #include <vector>
+#include <libintl.h>
 #include <klocale.h>
 #include <libintl.h>
 
diff -dur -x '*~' celestia-1.6.1.orig/src/celestia/kde/selectionpopup.cpp celestia-1.6.1/src/celestia/kde/selectionpopup.cpp
--- celestia-1.6.1.orig/src/celestia/kde/selectionpopup.cpp	2011-06-05 18:11:13.000000000 +0200
+++ celestia-1.6.1/src/celestia/kde/selectionpopup.cpp	2015-07-10 19:59:04.249679059 +0200
@@ -20,6 +20,7 @@
 #include <sstream>
 #include <fstream>
 #include <algorithm>
+#include <libintl.h>
 
 #include "selectionpopup.h"
 
diff -dur -x '*~' celestia-1.6.1.orig/src/celestia/oggtheoracapture.cpp celestia-1.6.1/src/celestia/oggtheoracapture.cpp
--- celestia-1.6.1.orig/src/celestia/oggtheoracapture.cpp	2011-06-05 18:11:13.000000000 +0200
+++ celestia-1.6.1/src/celestia/oggtheoracapture.cpp	2015-07-10 19:59:04.249679059 +0200
@@ -62,6 +62,9 @@
 #include <cstdlib>
 #include <cstdio>
 #include <cmath>
+#include <cstring>
+#include <ctime>
+#include <libintl.h>
 #include <celutil/debug.h>
 #include <celutil/util.h>
 #include "../celengine/gl.h"
diff -dur -x '*~' celestia-1.6.1.orig/src/celmath/intersect.h celestia-1.6.1/src/celmath/intersect.h
--- celestia-1.6.1.orig/src/celmath/intersect.h	2011-06-05 18:11:15.000000000 +0200
+++ celestia-1.6.1/src/celmath/intersect.h	2015-07-10 19:59:04.249679059 +0200
@@ -12,6 +12,7 @@
 #ifndef _CELMATH_INTERSECT_H_
 #define _CELMATH_INTERSECT_H_
 
+#include "mathlib.h"
 #include "ray.h"
 #include "sphere.h"
 #include "ellipsoid.h"
diff -dur -x '*~' celestia-1.6.1.orig/src/celutil/util.cpp celestia-1.6.1/src/celutil/util.cpp
--- celestia-1.6.1.orig/src/celutil/util.cpp	2011-06-05 18:11:15.000000000 +0200
+++ celestia-1.6.1/src/celutil/util.cpp	2015-07-10 19:59:04.249679059 +0200
@@ -12,6 +12,7 @@
 #include "util.h"
 #include <sys/types.h>
 #include <sys/stat.h>
+#include <libintl.h>
 
 using namespace std;
 
