diff -ruN celestia-1.4.0./celestia.cfg celestia-1.4.0/celestia.cfg
--- celestia-1.4.0./celestia.cfg	2005-09-04 23:35:51.000000000 +0200
+++ celestia-1.4.0/celestia.cfg	2005-12-25 01:33:24.169965072 +0100
@@ -137,7 +137,7 @@
 # or
 #   ExtrasDirectories  [ "D:\\celestia-extras" ]
 #------------------------------------------------------------------------
-  ExtrasDirectories  [ "extras" ]
+  ExtrasDirectories  [ "extras" "~/.celestia" ]
 
 
 #------------------------------------------------------------------------
