diff -ruN celestia-1.2.5-/data/solarsys.ssc celestia-1.2.5/data/solarsys.ssc
--- celestia-1.2.5-/data/solarsys.ssc	Tue Jan  7 16:36:56 2003
+++ celestia-1.2.5/data/solarsys.ssc	Tue Jan 14 01:45:22 2003
@@ -321,6 +321,12 @@
 	RotationOffset      305.40
 
 	Albedo 0.51
+
+	Rings {
+		Inner 92000
+		Outer 221000
+		Texture "jupiter-rings.png"
+	}
 }
 
 
@@ -850,6 +856,12 @@
 	RotationOffset       228.65
 
 	Albedo                 0.62
+
+	Rings {
+		Inner 41900
+		Outer 62983
+		Texture "neptune-rings.png"
+	}
 }
 
 "Proteus" "Sol/Neptune"
