diff -ruN celestia-1.4.0./src/celengine/command.h celestia-1.4.0/src/celengine/command.h
--- celestia-1.4.0./src/celengine/command.h	2004-12-22 23:07:15.000000000 +0100
+++ celestia-1.4.0/src/celengine/command.h	2005-12-25 00:40:12.545499680 +0100
@@ -94,7 +94,7 @@
 class CommandGotoLongLat : public InstantaneousCommand
 {
  public:
-    CommandGotoLongLat::CommandGotoLongLat(double t,
+    CommandGotoLongLat(double t,
                                            double dist,
                                            float _longitude, float _latitude,
                                            Vec3f _up);
diff -ruN celestia-1.4.0./src/celengine/star.h celestia-1.4.0/src/celengine/star.h
--- celestia-1.4.0./src/celengine/star.h	2005-12-06 18:30:23.000000000 +0100
+++ celestia-1.4.0/src/celengine/star.h	2005-12-25 00:40:20.201335816 +0100
@@ -30,7 +30,7 @@
     // orbits and StarDetails objects.  Currently, it's assumed that they
     // have lifetimes that continue until program termination.
  public:
-    StarDetails::StarDetails();
+    StarDetails();
 
     inline float getRadius() const;
     inline float getTemperature() const;
