diff -ruN celestia./configure.in celestia/configure.in
--- celestia./configure.in	2005-01-11 21:40:24.000000000 +0100
+++ celestia/configure.in	2005-01-11 21:40:46.963692448 +0100
@@ -359,7 +359,7 @@
 AC_MSG_CHECKING([whether to enable Celestia Extension Language])
 if (test "$enable_lua" != "no"); then
 	CXXFLAGS="$CXXFLAGS -DCELX"
-	LDFLAGS="$LDFLAGS -llualib -llua"
+	LDFLAGS="$LDFLAGS -llualib50 -llua50"
 	
 	if (test "$lua_includes" != "no"); then
 		CXXFLAGS="$CXXFLAGS -I$lua_includes"
